README.md

```diff
@@ -29,10 +29,19 @@ Please see the following for detailed descriptions of each type:
    ```
    ./generate-go.sh TARGET_PATH
    ```
+   
+### Java
+
+1. Build the go code first (see above) so that all the imports are available
```

Comment:
Is there a way we could make this step unnecessary? Requiring a Java developer to have Go and all of the associated tooling available seems less than optional.

On the other hand, I know nothing about the Java tooling, so maybe this really is necessary?